### ISSUE TITLE
eating ALL the pickles of a pickle jar turns it into a beaker, deleting coroner heirloom

### ISSUE THREAD

**Issue by carlarctg** [tgstation/tgstation#87449]
### Issue Summary:

eating ALL the pickles of a pickle jar turns it into a beaker, deleting coroner heirloom

### Reproduction Steps:

eating ALL the pickles of a pickle jar turns it into a beaker, deleting coroner heirloom

**Comment by SmArtKar** [comment]
Incorrect, it turns into a beaker once you use it in-hand to "fold" it - eating all the pickles out of it does not cause this.
